Assets/Scripts/Models/DialogBoxInformation/DialogComponent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+using UnityEngine;
+
+public class DialogComponent
+{
+    [XmlAttribute("Type")]
```

Comment:
according to guidelines here, we have elements starting with capitals and attributes are camelCase, so:
`[XmlAttribute("type")]`